game/scripts/npc/items/item_meteor_hammer.txt

```diff
@@ -30,22 +31,26 @@
   //=================================================================================================================
   // Meteor Hammer 1
   //=================================================================================================================
-  "item_meteor_hammer"
+  "item_meteor_hammer" "REMOVED"
+  "item_meteor_hammer_1"
   {
     // General
     //-------------------------------------------------------------------------------------------------------------
-    "ID"                            "223"                                                       // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
+    "ID"                            "4050"                                                       // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
+    "BaseClass"                     "item_lua"
+    "ScriptFile"                    "items/meteor_hammer.lua"
     "AbilityBehavior"               "DOTA_ABILITY_BEHAVIOR_POINT | DOTA_ABILITY_BEHAVIOR_AOE | DOTA_ABILITY_BEHAVIOR_CHANNELLED"
     "AbilityTextureName"            "custom/meteor_hammer_1"
     "AbilityUnitDamageType"         "DAMAGE_TYPE_MAGICAL"
     "SpellImmunityType"             "SPELL_IMMUNITY_ENEMIES_NO"
     "FightRecapLevel"               "1"
 
-    // Spicy Parameters
-    //-------------------------------------------------------------------------------------------------------------
+    // Spicy Paramaters
+    //-------------------------------------------------------------------------------------------------------------                                  
+    
     "MaxUpgradeLevel"                                     "5"
     "ItemBaseLevel"                                       "1"
-
+    "UpgradesItems"                                       "item_meteor_hammer_1;item_meteor_hammer_2;item_meteor_hammer_3;item_meteor_hammer_4;item_meteor_hammer_5"
```

Comment:
I believe `UpgradesItems` is unnecessary. It's for items like vanilla Dagon that upgrade by purchasing the same recipe multiple times. We have it in most of our items still because it wasn't known what it was for and haven't bothered to remove it.